code/modules/reagents/chemistry/reagents/misc_reagents.dm

```diff
@@ -502,7 +503,10 @@
 			"You feel like telling a pun.")
 			to_chat(M, "<span class='warning'>[pick(clown_message)]</span>")
 		if(M.mind.assigned_role == "Mime")
-			update_flags |= M.adjustToxLoss(1.5 * REAGENTS_EFFECT_MULTIPLIER)
+			if(!M.dna.species.tox_mod) // If they can't take tox damage, make them take burn damage
```

Comment:
```suggestion
			if(M.dna.species.tox_mod == 0) // If they can't take tox damage, make them take burn damage
```

Reply:
I think ! is better over ==, but this made me realize that a negative tox mod could heal.
```suggestion
			if(M.dna.species.tox_mod <= 0) // If they can't take tox damage, make them take burn damage
```